native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/sql-native/resource-config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "resources":{
```

Comment:
Lets format this json file

Reply:
Fixed in [7655a3a](https://github.com/ballerina-platform/module-ballerina-sql/pull/563/commits/7655a3a7e3e4de9e864c1cdc93eeedaad7952a61)